Add instructions for setting up git signing

mgwalker opened the pull request:
## Changes proposed in this pull request:

- Just what it says on the tin. This adds a new page to help folks get git signing setup.

mgwalker committed:
add instructions for setting up git signing

JJediny committed:
Update privateLinks.js

The repo should reference the org where allstar is running from

vrajmohan reviewed (commented):
Minor. TIL, KeePassXC has ssh integration.

nateborr reviewed (commented):
I followed the KeyPass instructions and confirmed that git is signing my commits and GitHub marks them as verified! The manual steps look good to me although I didn't walk through them.

I added a couple of small and non-blocking notes on content. It looks good in terms of technical accuracy and overall flow! I'm setting this review at "Comment" pending another readability review.

JJediny reviewed (dismissed):
thanks @mgwalker 💯 some minor comments but nothing that should block merging

mgwalker committed:
updates from feedback

ajfarkas reviewed (commented):
I think this is all fine language for TTS purposes. If you want this to be accessible to the broader public, I have two loosely held suggestions.

mgwalker committed:
feedback; add crypto algorithm, remove backup

mgwalker committed:
oop; typo

AnitaYCheng committed:
Content designed the instructions

Created lots and lots of numbered lists. And removed all the manual line breaks.

AnitaYCheng committed:
Replace tags with keywords

mgwalker committed:
remove prettier prose wrapping

JJediny reviewed (approved):
@mgwalker looks good to merge 💯
